pkg/admission/interfaces.go

```diff
@@ -64,6 +64,7 @@ type Operation string
 const (
 	Create  Operation = "CREATE"
 	Update  Operation = "UPDATE"
+	Patch   Operation = "PATCH"
```

Comment:
Then Patch is broken currently? PATCH and UPDATE are both logically a pair of (GET, SET) operations. Are you saying you want to admit the GET before you admit the SET?